gloo/docs/user_guides/basic_ingress.md

```diff
@@ -17,38 +17,63 @@ written with the annotation `kubernetes.io/ingress.class: gloo`.
  
 1. Next, deploy the Pet Store app to kubernetes:
 
-        kubectl apply \
-         -f https://raw.githubusercontent.com/solo-io/gloo/master/example/petstore/petstore.yaml
+kubectl apply \
+   -f https://raw.githubusercontent.com/solo-io/gloo/master/example/petstore/petstore.yaml
 
 1. Let's create a Kubernetes Ingress object to route requests to the petstore
 
-        cat <<EOF | kubectl apply -f -
-        apiVersion: extensions/v1beta1
-        kind: Ingress
-        metadata:
-         name: petstore-ingress
-         namespace: default
-         annotations:
-            kubernetes.io/ingress.class: gloo
-        spec:
-         rules:
-         - http:
-             paths:
-             - path: /.*
-               backend:
-                 serviceName: petstore
-                 servicePort: 8080
-        EOF
+{{< highlight shell >}}
+cat <<EOF | kubectl apply -f -
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+ name: petstore-ingress
```

Comment:
yaml indenting looks broken (from original). shouldn't it be 2 space indent? Good to test copy and paste of commands to make sure it works as expected.